RevenueCatUI/CustomerCenter/Views/UIKit Compatibility/CustomerCenterViewController.swift

```diff
@@ -33,23 +33,176 @@ import SwiftUI
 @available(watchOS, unavailable)
 public class CustomerCenterViewController: UIHostingController<CustomerCenterView> {
 
+    // Handler for when the view controller is dismissed
+    public var onDismiss: (() -> Void)?
+
+    // Stored handlers for later updating if needed
+    private var restoreStartedHandler: CustomerCenterView.RestoreStartedHandler?
+    private var restoreCompletedHandler: CustomerCenterView.RestoreCompletedHandler?
+    private var restoreFailedHandler: CustomerCenterView.RestoreFailedHandler?
+    private var showingManageSubscriptionsHandler: CustomerCenterView.ShowingManageSubscriptionsHandler?
+    private var refundRequestStartedHandler: CustomerCenterView.RefundRequestStartedHandler?
+    private var refundRequestCompletedHandler: CustomerCenterView.RefundRequestCompletedHandler?
+    private var feedbackSurveyCompletedHandler: CustomerCenterView.FeedbackSurveyCompletedHandler?
+
+    // Legacy handler for backward compatibility
+    private var legacyHandler: DeprecatedCustomerCenterActionHandler?
```

Comment:
Instead of storing both the legacy handler and the individual handlers, I believe we could map the legacy handler into the individual ones, similar to what's done in [RevenueCatUI/CustomerCenter/View+PresentCustomerCenter.swift](https://github.com/RevenueCat/purchases-ios/blob/0dc7f065bf274f76b88d3ad6efce00a4a1f03301/RevenueCatUI/CustomerCenter/View%2BPresentCustomerCenter.swift#L73)

Not a big deal though. So FFTI